code/game/machinery/vending.dm

```diff
@@ -1814,6 +1814,7 @@
 					/obj/item/clothing/head/officer = 4,
 					/obj/item/clothing/head/beret/sec = 4,
 					/obj/item/clothing/head/soft/sec = 4,
+					/obj/item/clothing/mask/bandana/red = 4,
```

Comment:
Might need to call on nasa for a compute farm with enough power to review this diff.

Reply:
so much power the Bot is MAD.